### ISSUE TITLE
Stuck at Connecting to DockerHub when trying to download an image in the public hub.

### ISSUE THREAD

**Issue by djsly** [kitematic/kitematic#475]
when when I look with docker images, it is actually downloading the images

CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              13fae5cd4562        6 months ago        413.4 MB
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              13fae5cd4562        6 months ago        413.4 MB
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              13fae5cd4562        6 months ago        413.4 MB
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              3e49dc90ad50        6 months ago        430.1 MB
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              9a9f248bc1c5        6 months ago        430.1 MB
bash-3.2$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
<none>              <none>              9a9f248bc1c5        6 months ago        430.1 MB

![screen shot 2015-05-04 at 22 27 53](https://cloud.githubusercontent.com/assets/4981802/7466214/fc8fd784-f2ac-11e4-8a91-361feafe8069.png)

Messing / progress should be shown.

Version 0.5.13

**Comment by mchiang0610** [comment]
@djsly Sorry for the delay. Can you try this again? This is most likely a one time issue with Docker Hub API. 

It will usually correct itself. (Just tested on my end). 

Closing this issue for now. Feel free to reopen this or reply if you have more questions.

Thanks!

**Comment by keganpowers34** [comment]
is there anyway to fix this? i'm having the same problem

**Comment by bedwar2** [comment]
> is there anyway to fix this? i'm having the same problem

Me too, shut down a few times.  Just trying to create hello-world-nginx

**Comment by DebasishOpu** [comment]
Having the same problem for Splash!

**Comment by twixi** [comment]
win 7 x64 i have too this...

**Comment by ser-storchak** [comment]
Windows Home 10.0.18363 x64 i have too this...

**Comment by pinhaocheng** [comment]
Hey y'all, this works for me:

-Windows 10 Home
-Docker toolbox
-Kinematics
-Splash

0. open up kinematic
1. start PowerShell by clicking the button on the lower left "Docker Cli" button
2. type in command `docker pull scrapinghub/splash`
3. the attachments should start downloading and status bar will be shown

*the first time I ran step 2 it showed some network handshaking error, but it started downloading once I ran the same command again.
